LayoutTests/fast/canvas/canvas-pattern-from-modified-canvas.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
+<html>
+<head>
+<script src="../../resources/js-test-pre.js"></script>
+</head>
+<body>
+<script>
+description("Test to ensure that modifications to canvas are not visible from already captured patterns.");
```

Comment:
Is this bug repro only when GPUProcess for DOM rendering is disabled? I think this bug is related to the case when RemoteImageBufferProxy::canMapBackingStore() returns true. 

If this is the case, please indicate in the title and the description that this bug is only repro on macOS.